docs_src/use-cases/automated-self-checkout/catalog/Test-Performance.md

```diff
@@ -0,0 +1,50 @@
+# Test Performance
+
+You can run the performance tests across CPU and GPU, and across all architectures, in combination and individually to provide flexibility in identifying performance needs.
+
+You can collect the metrics for predefined pipelines or for stream density. 
+You can run the script in two ways:
+
+* Defined Pipelines: This allows you to pre-define the number of pipelines and gather the performance metrics.
+* Stream Density: This allows you to pre-define the target frames per second (FPS) rate required per pipeline, and then measure metrics.   
+
+## Benchmark Predefined Pipelines
```

Comment:
Do we need instructions on getting the performance-tools submodule? or does that get downloaded elsewhere?

Reply:
I removed this page and added a link to the full performance benchmark documentation.